README.md

```diff
@@ -1,16 +1,189 @@
-# Title
+# Ruby Method Arguments
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe the functionality of arguments
+- Define methods that accept arguments
+- Recognize the difference between arguments and parameters
 
 ## Introduction
 
-## SWBAT 1
+In the real world, many interactions are customized or specific to our
+day-to-day experiences. For example, if you go to a coffee shop, they may take
+down your name to call up your order when it's ready. If visit a doctor's
+office, they may call your name when it's your turn to be seen. Variables that
+we experience in real life often need to be translated into to programs.
 
-## SWBAT 2
+In the example of the doctor's office or the coffee shop they have a "template"
+behavior that they apply to everyone, but that they change only slightly
+depending on the particular customer or patient. Methods can also be written to
+have a "template" of shared activity but which can be changed just a little bit
+depending on some particular data. The particular bits that make the change are
+called "arguments."
+
+## Describe the functionality of arguments
+
+When we log into email, social media, or any other user-based platform, what do
+we normally see after logging in?
+
+Our name somewhere on the account, right?
+
+So far, we've only discussed how to create methods that return static
+information. For example:
+
+```ruby
+def greeting
+  puts "Hi, Ruby programmer!"
+end
+
+greeting
+#=> Hi, Ruby Programmer
+```
+
+As amazing as this method is, it's still pretty static. It hard-codes, or
+directly specifies, the name of the person we are greeting as `"Ruby
+programmer"`. If we wanted to build a method that can greet _anyone_ by name,
+we'd need to be able to add input into the body of the method.
+
+Just like with a real-world greeting, we'd want our method to be more dynamic,
+more abstract, and _more re-usable_. Remember our DRY (Don't Repeat Yourself)
+principle? The method should maintain the elements that will always be the same,
+no matter who we greet, but allow us to change, or swap out, the name of the
+person we are greeting. This is "dynamic", as opposed to "hard-coded".
+
+## Define methods that accept arguments
+
+Now that we see the limitations of basic methods, how do we take the information
+we know about arguments and apply it to real-world scenarios like that of a
+doctor's office or a coffee shop?
+
+To add arguments to a method, you specify them in he line that starts with
+`def`. 
+
+For example, if we want to write a method called `greeting_a_person` that
+accepts an argument of a person's name, we would do it like this:
+
+```ruby
+    #method name      #argument
+def greeting_a_person(name)
+  "Hello #{name}"
+end
+```
+
+Arguments create new _local_ variables that can be used within the method. When
```

Comment:
Parameters?

Reply:
within the _scope_ of the method. make explicit the tie to why we sequence as we do.